cicd/42c-conf-examples/mapping-no-discovery/42c-conf.yaml

```diff
@@ -2,18 +2,15 @@
 # only audit the API files you have specifically mapped to existing 
 # APIs in 42Crunch Platform.
 #
-# Discovery phase is enabled by default, otherwise the CI/CD integration
-# plugin would not find any API files to audit. However, if you map the
-# filenames of the API definitions that you want to include to
-# the API UUIDs in the platform, you can exclude any other files.
-#
-# When the plugin starts, it begins with cleaning the 
-# API collection you have defined it to use. That is why the mapping 
-# must be done to another API collection.
+# When the CI/CD plugin starts, it creates an API collection
+# using the source control repository name and branch name.
 #
 # The plugin uploads any APIs it finds during the discovery phase
-# to 42Crunch Platform, so the APIs always get new API UUIDs. Mapping is also 
-# useful if you want to preserve API UUIDs across separate audit runs.  
+# into this API collection. During the subsequent runs the collection
+# is kept is sync with the APIs discovered in your sources.
+#
+# Discovery is enabled by default. However, it can be disabled if you 
```

Comment:
"However, it can be disabled if you want to not use discovery at all and instead to rely on mapping." --> "However, you can switch it off and rely solely on mappings if you do not want to use discovery."